src/components/accession2/viabilityTesting/NewViabilityTestModal.tsx

```diff
@@ -142,8 +142,8 @@ export default function NewViabilityTestModal(props: NewViabilityTestModalProps)
       record?.testResults.forEach((_tr, index) => {
         setIndividualError(`seedsGerminated${index}`, '');
       });
-      return true;
     }
+    return true;
```

Comment:
i maybe contributed to this error in the last PR - when I asked to invert the boolean result of all the validateXYZ functions.

Reply:
Not sure, but thanks for fixing it! :)